app/services/support_interface/work_history_break_export.rb

```diff
@@ -0,0 +1,223 @@
+module SupportInterface
+  class WorkHistoryBreakExport
+    class UnexplainedBreak
+      def initialize(month_range:)
+        @month_range = month_range
+      end
+
+      def start_date
+        @month_range.first.prev_month.in_time_zone
+      end
+
+      def end_date
+        @month_range.last.next_month.in_time_zone
+      end
+
+      def length
+        ((end_date.year * 12) + end_date.month) - ((start_date.year * 12) + start_date.month)
+      end
+    end
+
+    def data_for_export
+      applications = ApplicationForm
+                         .where.not(date_of_birth: nil)
+                         .select(:id, :candidate_id, :submitted_at, :date_of_birth, :work_history_completed)
+                         .includes(:application_qualifications, :application_work_experiences, :application_work_history_breaks, :application_volunteering_experiences, :application_choices)
+                         .order(submitted_at: :desc).uniq(&:candidate_id)
```

Comment:
Consider extracting this to its own method `applications` (or maybe `application_forms` method for clarity), so the focus in this method is exporting the necessary data.

Reply:
Done